### PR TITLE
MM-23873: Add profile handlers

### PR DESCRIPTION
**Pull request by agnivade**
#### Summary

- Reset the MemStore on various occasions.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-23873

### CODE REVIEW

File: loadtest/control/simulcontroller/controller.go [hunk 1]
```diff
@@ -58,6 +58,7 @@ func (c *SimulController) Run() {
 			c.status <- c.newErrorStatus(err)
 		}
 		c.user.Cleanup()
+		c.user.ClearUserData()
```
Comment (streamer45) [inline loadtest/control/simulcontroller/controller.go:61]: That's an interesting one. Is this actually needed or does it simply help the GC to re-collect the memory more quickly?

File: loadtest/control/simplecontroller/controller.go [hunk 2]
```diff
@@ -154,7 +155,14 @@ func (c *SimpleController) createActions(definitions []actionDefinition) error {
...
+			resp := control.Logout(u)
+			if resp.Err != nil {
+				return resp
+			}
+			u.ClearUserData()
+			return resp
+		},
 		"EditPost": control.EditPost,
 		"Reload": func(u user.User) control.UserActionResponse {
 			return c.reload(false)
```
Comment (streamer45) [inline loadtest/control/simplecontroller/controller.go:168]: Can we also fix this since we are here? It should be called with `true`.

Comment (agnivade) [reply loadtest/control/simplecontroller/controller.go:168]: Sure

File: loadtest/control/simulcontroller/controller.go (hunk 1 above)
Comment (agnivade) [reply loadtest/control/simulcontroller/controller.go:61]: This is needed in cases when the coordinator stops an agent, and then starts it again. So the agent never shuts down, and unless the store is cleared, will leak memory.

Comment (agnivade) [reply loadtest/control/simulcontroller/controller.go:61]: Oh you mean the entire controller struct goes out of scope, is it ? Yeah then I guess it's not needed. But maybe good to add it anyways.

Comment (streamer45) [reply loadtest/control/simulcontroller/controller.go:61]: Yeah, fine with keeping it. Just thought that when we are done with the controller we remove it from the slice and the GC should do the rest.

**Pull request merged**